mr/src/test/java/org/apache/mahout/classifier/df/tools/VisualizerTest.java

```diff
@@ -17,9 +17,7 @@
 
 package org.apache.mahout.classifier.df.tools;
 
-import java.util.List;
-import java.util.Random;
-
+import com.google.common.collect.Lists;
```

Comment:
Not sure why this PR still says 2 commits, there should be 3 now. Can you take the third commit manually?